src/style.css

```diff
@@ -1,3 +1,82 @@
-body {
+* {
+  list-style: none;
+  text-decoration: none;
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --link-color: #000;
+  --hover-link-color: #00acdc;
+  --nav-link-fz: 12px;
+  --header-line-height: 60px;
+  --header-logo-image: url(./images/logo.png);
+}
+
+body {
+  list-style: none;
+  text-decoration: none;
+}
+
+.header {
+  margin: 0 auto;
+  font: var(--nav-link-fz) Roboto, sans-serif;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  line-height: var(--header-line-height);
```

Comment:
I would say it is not the correct place to set line-height
Better move it to more specific selectors like `nav__link`